packages/api-v2/src/student/student.controller.ts

```diff
@@ -71,6 +72,31 @@ export class StudentController {
     return student;
   }
 
+  @UseGuards(JwtAuthGuard)
```

Comment:
i don't remember: does this guard make sure that the logged in user matches their uuid?

Reply:
this guard simply ensures that the JWT is valid and that the uuid stored in the JWT is valid. It then fetches the student corresponding to the uuid stored in the JWT and adds it to the request.